spec/forms/user_phone_form_spec.rb

```diff
@@ -6,62 +6,13 @@
   let(:user) { build(:user, :signed_up) }
   let(:params) do
     {
-      phone: '703-555-5000',
-      international_code: 'US',
       otp_delivery_preference: 'sms',
     }
   end
-  subject { UserPhoneForm.new(user, MfaContext.new(user).phone_configurations.first) }
-
-  it_behaves_like 'a phone form'
-
-  it 'loads initial values from the user object' do
-    user = build_stubbed(
-      :user, :with_phone,
-      with: { phone: '+1 (703) 500-5000' },
-      otp_delivery_preference: 'voice'
-    )
-    subject = UserPhoneForm.new(user, MfaContext.new(user).phone_configurations.first)
-
-    expect(subject.phone).to eq(MfaContext.new(user).phone_configurations.first.phone)
-    expect(subject.international_code).to eq('US')
-    expect(subject.otp_delivery_preference).to eq(user.otp_delivery_preference)
-  end
-
-  it 'infers the international code from the user phone number' do
-    user = build_stubbed(:user, :with_phone, with: { phone: '+81 744 21 1234' })
-    subject = UserPhoneForm.new(user, MfaContext.new(user).phone_configurations.first)
-
-    expect(subject.international_code).to eq('JP')
-  end
-
-  describe 'phone validation' do
-    it do
-      should validate_inclusion_of(:international_code).
-        in_array(PhoneNumberCapabilities::INTERNATIONAL_CODES.keys)
-    end
-
-    it 'validates that the number matches the requested international code' do
```

Comment:
Some of these tests still look useful. What is the reason for removing them?

Reply:
Removed by mistake. Re-added.